src/main/java/seedu/timetable/Timetable.java

```diff
@@ -16,65 +16,65 @@
 
```

Comment:
based on gson, will save earliestHour and latestHour in the json file. Might have some confusing since the I'm guessing this is for each day, but the timetable contains mon-fri

Reply:
earliestHour and latestHour is across all days because the timetableUI resizes what to print based on the earliest class and the latest class

Reply:
hmm couldn't we just make the timetable UI standard size, ie show from 0600-1700 like nusmods?
or isit cuz in the future might add timeslots way past those times
might be a bit jarring if timetable scale keeps changing

Reply:
So actually nusmods default is 1000 -1700, but stretches if classes are added to timeslots beyond or before. In my case I am not exactly rescaling the output, but rather simply extending it (fixed-size width timeslots, but number of timeslots change)

Currently my implemented default is 0900-1200, but I could set it to 0900-1600 so theres not so it doesn't change that often